homeassistant/components/reolink/sensor.py

```diff
@@ -60,6 +62,40 @@ class ReolinkHostSensorEntityDescription(
         value=lambda api, ch: api.ptz_pan_position(ch),
         supported=lambda api, ch: api.supported(ch, "ptz_position"),
     ),
+    ReolinkSensorEntityDescription(
+        key="battery_percent",
+        cmd_key="GetBatteryInfo",
+        translation_key="battery_percent",
```

Comment:
This translation key can be omitted to use the device class icon and name

Reply:
Follow up PR is here: https://github.com/home-assistant/core/pull/117616